src/main/java/com/commercetools/sync/types/utils/FieldDefinitionUpdateActionUtils.java

```diff
@@ -0,0 +1,106 @@
+package com.commercetools.sync.types.utils;
+
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.types.EnumFieldType;
+import io.sphere.sdk.types.FieldDefinition;
+import io.sphere.sdk.types.LocalizedEnumFieldType;
+import io.sphere.sdk.types.Type;
+import io.sphere.sdk.types.commands.updateactions.ChangeFieldDefinitionLabel;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import static com.commercetools.sync.commons.utils.CommonTypeUpdateActionUtils.buildUpdateAction;
+import static com.commercetools.sync.types.utils.LocalizedEnumUpdateActionUtils.buildLocalizedEnumValuesUpdateActions;
+import static com.commercetools.sync.types.utils.PlainEnumUpdateActionUtils.buildEnumValuesUpdateActions;
+import static java.util.stream.Collectors.toList;
+
+
+public final class FieldDefinitionUpdateActionUtils {
+
+    /**
+     * Compares all the fields of old {@link FieldDefinition} with new {@link FieldDefinition} and returns
+     * a list of {@link UpdateAction}&lt;{@link Type}&gt; as a result. If both the {@link FieldDefinition}
+     * and the {@link FieldDefinition} have identical fields, then no update action is needed and hence an
+     * empty {@link List} is returned.
+     *
+     * @param oldFieldDefinition the old field definition which should be updated.
+     * @param newFieldDefinition the new field definition where we get the new fields.
+     * @return A list with the update actions or an empty list if the field definition fields are identical.
+     */
+    @Nonnull
+    public static List<UpdateAction<Type>> buildActions(
+        @Nonnull final FieldDefinition oldFieldDefinition,
+        @Nonnull final FieldDefinition newFieldDefinition) {
+
+        final List<UpdateAction<Type>> updateActions = Stream.of(
+                buildChangeLabelUpdateAction(oldFieldDefinition, newFieldDefinition))
+            .filter(Optional::isPresent)
+            .map(Optional::get)
```

Comment:
Instead of using `.filter(Optional::isPresent).map(Optional::get)` you can now use the new util: https://github.com/commercetools/commercetools-sync-java/pull/307 😊

Reply:
Don't forget to do it here too: `TypeSyncUtils#buildActions` 😉